File: javascript/programmers/javascript/level2/멀쩡한사각형.js
function gcd(a, b) {
  while (b > 0) {
    let temp = b;
    b = a % b;
    a = temp;
  }
  return a;
}

function solution(w, h) {
  let answer = w * h;
  answer -= w + h - gcd(w, h);
  return answer;
}

File: python/programmers/level2/멀쩡한 사각형.py
def gcd(a, b):
    while b > 0:
        tmp = b
        b = a % b
        a = tmp
    return a


def solution(w, h):
    answer = w * h
    answer -= w + h - gcd(w, h)
    return answer


print(solution(8, 12))
